content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -314,3 +314,4 @@ For _Mounted Disk_ installations, you must:
     semanage fcontext -a -t container_file_t "/opt/tfe(/.*)?"
     restorecon -R /opt/tfe
     ```
+-> ** Note: ** The `install.sh` script will attempt to install `container-selinux` in online mode. Regradless of mode container-selinux package as a dependacy on the `policycoreutils-python` package being installed.
```

Comment:
```suggestion
-> **Note:** The online installer will attempt to install the `container-selinux` if it is not already installed.
```

Reply:
the install of `container-selinux ` will fail under RHEL as the installation method does not install `policycoreutils-python` which is a dependency on the OS of the container and not handled by the installer, `policycoreutils-python` is not part of the EPEL or RHEL extras standard packages , it is a package in Centos base  
the important part of the call out is the requirement for `policycoreutils-python`